testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2429,6 +2430,66 @@ protected void verifyAddNotNullColumnToNonEmptyTableFailurePermissible(Throwable
         throw new AssertionError("Unexpected adding not null columns failure", e);
     }
 
+    @Test
+    public void testAddRowField()
```

Comment:
nit
This is part of "Support adding a field with ADD COLUMN in engine".
At this point, no connector supports the new operation so the test is dead code. You can move BCT to the second commit.